### DISCUSSION TITLE
Failed to resolve: ai.coqui:libstt:0.9.3

### DISCUSSION THREAD

**Discussion by redaready** [coqui-ai/STT-examples#3] [General]
Hi, great project for asr, thanks!

when I open android_mic_streaming example in android studio, the project can't be build. error message: "Failed to resolve: ai.coqui:libstt:0.9.3"

am I missing something?

**Comment by soudalan** [comment]
Hi there! I am also getting this error in Android Studio. I tried all of the branches, same issue.
